app/views/admin/classrooms/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<%#
+# Show
+
+This view is the template for the show page.
+It renders the attributes of a resource,
+as well as a link to its edit page.
+
+## Local variables:
+
+- `page`:
+  An instance of [Administrate::Page::Show][1].
+  Contains methods for accessing the resource to be displayed on the page,
+  as well as helpers for describing how each attribute of the resource
+  should be displayed.
+
+[1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Show
+%>
+
+<% content_for(:title) { t("administrate.actions.show_resource", name: page.page_title) } %>
+
+<header class="main-content__header" role="banner">
+  <h1 class="main-content__page-title">
+    <%= content_for(:title) %>
+  </h1>
+
+  <div>
+    <% if accessible_action?(page.resource, :edit) %>
+      <%= link_to(
+        t("administrate.actions.edit_resource", name: page.page_title),
+        [:edit, namespace, page.resource],
+        class: "button",
+      ) %>
+    <% end %>
+
+    <% if page.resource.archived? %>
+      <%= button_to(
+        "Unarchive",
+        [:archive, namespace, page.resource],
+        class: "button button--success",
+        method: :patch,
+        data: { turbo_confirm: "Are you sure you want to unarchive this classroom?" }
+      ) %>
+    <% else %>
+      <%= button_to(
+        "Archive",
+        [:archive, namespace, page.resource],
+        class: "button button--danger",
+        method: :patch,
+        data: { turbo_confirm: "Are you sure you want to archive this classroom?" }
+      ) %>
+    <% end %>
+  </div>
+</header>
+
+<section class="main-content__body">
+  <dl>
+    <% page.attributes.each do |attribute| %>
```

Comment:
I think it has a different API 🤔 

`page.attributes.each do |title, attributes|`